### PR TITLE
Add interactivity

### PR DESCRIPTION
**Pull request by DanaSemaniv**

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from find_the_description_game import find_the_description
+
+
+games = [
+    find_the_description,
+]
```
Comment (TarasRudnyk) [inline config.py]: PEP8

File: main.py [hunk 2]
```diff
@@ -1,7 +1,24 @@
-from find_the_description_game import find_the_description
-from helpers import get_patterns
+from config import games
+from helpers import get_patterns, validate_input
 
 
+def print_games(games):
+    print('\nEnter the number of game you want to play:\n')
+
+    for game in range(len(games)):
```
Comment (TarasRudnyk) [inline main.py]: for game_number

**Review by TarasRudnyk — CHANGES_REQUESTED** [review]

File: config.py (hunk 1 above)
Comment (DanaSemaniv) [reply config.py]: fixed

File: main.py (hunk 2 above)
Comment (DanaSemaniv) [reply main.py]: fixed

**Review by TarasRudnyk — APPROVED** [review]

**Pull request merged**